.github/workflows/deploy-apis-to-production.yml

```diff
@@ -1574,5 +1573,6 @@ jobs:
 
       - name: Apply ingress changes
         run: |
+          kubectl apply -f k8s/nginx-ingress/prod-nginx-cm.yaml
```

Comment:
[add configmap updates in nginx workflow](https://github.com/airqo-platform/AirQo-api/pull/1011/commits/8fda839b6f4d0fe0a98956dbc1b12113e6389934)